src/test/resources/features.SCAWrapper/local/SCAWrapperFrontPageLocal.feature

```diff
@@ -1,6 +1,10 @@
 @suite
 Feature: Wrapper integrating with sca-frontend
 
+  Background:
+   # Given mongoDB is dropped
+    And A message is updated in reactive mongo
```

Comment:
Can this be updated to something like "And a message is posted to the messages API"?
Also And -> Given?

Reply:
this cant be done i have tried this but multiple message got publish which is equal to number of scenario hence I have removed this and added only to the message scenario